code/utils_split.py

```diff
@@ -0,0 +1,139 @@
+"""
+DeepSlide
+Splits the data into training, validation, and testing sets.
+
+Authors: Jason Wei, Behnaz Abdollahi, Saeed Hassanpour
+
+Last Modified: November 30, 2019 (Joseph DiPalma)
```

Comment:
This `Last Modified` is probably too hard to maintain and i think it should be removed from all files. 
This was before I knew `git blame` existed.

Reply:
@jasonwei20 I didn't know about `git blame`.  It's a much better solution.